static/js/classroom/game.js

```
@@ -0,0 +1,566 @@
+// Create the game instance
+const game = new Phaser.Game(config);
+
+// Game variables
+let player;
+let cursors;
+let interactKey;
+let interactButton;
+let interactText;
+let interactableObjects = [];
+let chairsOccupancy = {};
+let isNearInteractable = false;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'isNearInteractable' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)